daemon/start.go

```diff
@@ -178,6 +180,9 @@ func (daemon *Daemon) containerStart(container *container.Container, checkpoint
 		return fmt.Errorf("%s", errDesc)
 	}
 
+	containerActions.WithValues("start").UpdateSince(start)
+	containerCounter.WithValues("start").Inc()
```

Comment:
This is probably redundant with containerActions, as presuming that's a summary or histogram it already tracks this as the _count.